test/functional/wallet_bumpfee.py

```diff
@@ -176,7 +188,6 @@ def test_bumpfee_with_descendant_fails(rbf_node, rbf_node_address, dest_address)
     rbf_node.sendrawtransaction(tx["hex"])
     assert_raises_rpc_error(-8, "Transaction has descendants in the wallet", rbf_node.bumpfee, parent_id)
 
-
```

Comment:
Nit: avoid dropping or adding whitespace in code that you're not touching.

Reply:
fixed